sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -1689,7 +1710,9 @@
               "dir": "Current Year Income - Supporting Documents",
               "customClass": "font-weight-bold",
               "validate": {
-                "required": true
+                "required": true,
+                "customMessage": "You must upload at least one file, and the total number of uploaded files cannot exceed {{data.maxUploadedFiles}}.",
+                "custom": "const files = data[instance.key] || []; valid = files.length > data.maxUploadedFiles ? false : true;"
```

Comment:
Minor and not a blocker, can we check if one of the below could be used instead of `data[instance.key]`?

![image](https://github.com/user-attachments/assets/767dcd31-01fd-4307-a1e7-c81588bc8798)

Reply:
For this component input seems to be every item when they are uploaded and value is undefined:
![image](https://github.com/user-attachments/assets/1ddbb9fb-1f18-421a-bf34-21a4822d7827)